src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +29,9 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+
+        logger.info("successful fetched data from DB");
```

Comment:
The logging message 'successful fetched data from DB' is misleading since the method currently returns dummy data and does not interact with a database. Consider updating the message to reflect the current functionality, such as 'Fetched dummy data for products'.

Reply:
that was a task // TODO: add log message about successful fetched data from DB